GI_App_Project/Time/Time_sellectView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Time_sellectView.swift
+//  Gi
+//
+//  Created by Daol on 2023/06/22.
+//
+import SwiftUI
+
+struct Time_sellectView: View {
+    @State private var tabSelection = 1
+    
+    var body: some View {
+        ZStack() {
+            TabView(selection: $tabSelection) {
+                Time_HyehwaBox()
+                    .tag(1)
+                Time_Alpha1Box()
+                    .tag(2)
+                Time_Alpha2Box()
+                    .tag(3)
+            }
+            .tabViewStyle(.page(indexDisplayMode: .always))
+            .frame(width: 393, height: 430)
```

Comment:
```suggestion
            .frame(width: UIScreen.main.bounds.size.width, height: 430)
```
frame width를 화면 너비와 동일하게 주고 싶다면, 다양한 디바이스에서도 대응하기 위해 해당 변수를 사용해주세요.